tests/unit/ai/test_tui.py

```
@@ -0,0 +1,94 @@
+        # Simulate some work
+        pass
+
+
+def test_ai_progress_exception_handling():
+    """Test ai_progress properly handles exceptions."""
+    mock_console = Mock(spec=Console)
+    mock_status = Mock()
+    mock_console.status.return_value.__enter__ = Mock(return_value=mock_status)
+    mock_console.status.return_value.__exit__ = Mock(return_value=None)
+
+    with pytest.raises(ValueError), ai_progress(mock_console, "Test"):
```

Ruff (0.14.7)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)